lib/reactTags.js

```diff
@@ -168,6 +167,11 @@ var ReactTags = React.createClass({
             });
         }
     },
+    handlePaste: function (e) {
+      e.preventDefault()
+      let string = e.clipboardData.getData('Text')
```

Comment:
In IE11 this causes an error. We need to check for `e.clipboardData` and if it doesn't exist get it from `window` (where IE11 keeps it).

```js
const clipboardData = e.clipboardData || window.clipboardData
const string = clipboardData.getData('text')
```

Also note that lowercase "text" seems to be the standard.